app/main.py

```diff
@@ -1,3 +1,48 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        return Distance(km=self.km + other.km) \
+            if isinstance(other, Distance) else Distance(km=self.km + other)
```

Comment:
It's better to avoid using `\` as a linebreaker. You can use regular conditions instead of ternary operator

Reply:
quite strange to see this suggestion and at the same time use comprehension